R/stats-glm-tidiers.R

```diff
@@ -5,13 +5,20 @@
 #' @template param_confint
 #' @template param_exponentiate
 #' @template param_unused_dots
+#' @param conf.method Controls whether stats::confint.default (Wald method)
+#'   or stats::confint (Profile likelihood method) will be used to compute
+#'   the coefficients confidence intervals. The former is typically faster.
+#'   The allowed types are `"plik"` for profile likelihood or `"wald"`
```

Comment:
Let's change `plik` to `profile`. Whole words are the preferred style in `broom`.